src/Microsoft.AspNet.Hosting/Startup/StartupLoader.cs

```diff
@@ -126,7 +126,7 @@ private static MethodInfo FindMethod(Type startupType, string methodName, string
             {
                 if (required)
                 {
-                    throw new InvalidOperationException(string.Format("A method named '{0}' or '{1}' in the type '{2}' could not be found.",
+                    throw new InvalidOperationException(string.Format("A public method named '{0}' or '{1}' in the type '{2}' could not be found.",
```

Comment:
This message sounds bad. It should be "A public method named ... could not be found in the ... type"